documentation/parent-child.html

```diff
@@ -85,14 +87,14 @@ <h2 id="parent-or-multi-values">Parent or multi-value?</h2>
 As a rule of thumb, model the items searched for as the document - example products for sale.
 Shared properties, like vendor, can be model using a parent document, importing a vendor name field -
 assuming a vendor has many products, and the vendor list is limited.
-Use arrays of structs for properties documents might have, like shoesize or screen resolution -
+Use arrays or maps of structs for properties documents might have, like shoesize or screen resolution -
 one can then have a struct field for property name and another for property value,
 giving a flexible structure for products with an unlimited set of possible properties.
 </p>
 
 
 
-<h2 id="example">Example</h2>
+<h2 id="example">Parent/child example</h2>
```

Comment:
change id attribute to _parent-child-example_ as well (same as heading), making sure to change possible links to it, too - ids and headings shall map 1-1